src/sentry/api/endpoints/event_attachment_details.py

```diff
@@ -56,16 +56,22 @@ class EventAttachmentDetailsEndpoint(ProjectEndpoint):
     permission_classes = (EventAttachmentDetailsPermission,)
 
     def download(self, attachment):
-        fp = attachment.getfile()
         name = posixpath.basename(" ".join(attachment.name.split()))
 
-        response = FileResponse(
-            fp,
-            as_attachment=True,
-            filename=name,
+        def stream_attachment():
+            with attachment.getfile() as fp:
+                while True:
+                    chunk = fp.read(4096)
+                    if not chunk:
+                        return
```

Comment:
https://docs.python.org/3/reference/expressions.html#assignment-expressions

```suggestion
                while chunk := fp.read(4096):
```